app/components/second/interrogation-content/interrogation-form/utils.ts

```diff
@@ -0,0 +1,13 @@
+export  const presentInitialValue = (inviteInfo: any, paramName: string) => {
+    let resultValue;
+    if (!inviteInfo.surveyResponses?.[paramName]) {
+        return ''
+    }
+    if (!inviteInfo.surveyResponses?.[paramName].length) {
+        resultValue = 'no';
+    } else {
+        resultValue = inviteInfo.surveyResponses?.[paramName].length === inviteInfo.guests.length ? 'yes' : 'any';
+    }
+
+    return resultValue
```

Comment:
you don't need let variable
```suggestion
    if (!inviteInfo.surveyResponses?.[paramName]) {
        return ''
    }
    
    if (!inviteInfo.surveyResponses?.[paramName].length) {
       return 'no';
    }
    
    return  inviteInfo.surveyResponses?.[paramName].length === inviteInfo.guests.length ? 'yes' : 'any';
 ```